src/components/App.js

```diff
@@ -3,13 +3,30 @@ import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 import './App.css';
 // eslint-disable-next-line no-unused-vars
-import calculate from '../logic/operate';
+import calculate from '../logic/calculate';
 
-const App = () => (
-  <div className="App">
-    <Display />
-    <ButtonPanel />
-  </div>
-);
+const App = () => {
+  const [state, setState] = React.useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
```

Comment:
**[ REQUIRED ]** As the scope of the project requires, we need to make our App component stateful, I can see you used the useState hook for this, this while it gets the job done, it is not within the scope of this project, as you can see [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896095/) 
![image](https://user-images.githubusercontent.com/46115725/72089438-a7068f00-32d1-11ea-8ef2-4030683a1131.png)

Let us make sure our App is a stateful component and you are good to go.

Reply:
the changes were done.